geomstats/geometry/stiefel.py

```diff
@@ -94,8 +94,8 @@ def inner_product(self, tangent_vec_a, tangent_vec_b, base_point):
 
         References
         ----------
-        .. [1] Formula from:
-        http://noodle.med.yale.edu/hdtag/notes/steifel_notes.pdf
+        .. [sn] Formula from:
+          http://noodle.med.yale.edu/hdtag/notes/steifel_notes.pdf
```

Comment:
Web links like this are generally discouraged in docstrings. Maybe we can find a proper reference.

Reply:
Right. For formulas (like here), we can write the formula in latex.
For references, we can add the full reference, with the link to the .pdf which is less likely to get deprecated than just a weblink.

Reply:
[This](https://epubs.siam.org/doi/abs/10.1137/16M1074485?journalCode=sjmael) one ought to do for instance (cf. the equation right after (2.1) in the [preprint](https://arxiv.org/pdf/1604.05054.pdf)).

Reply:
Yes I agree. This will be one of our big todo's in the docstring cleanup.

Reply:
Is this better?
![image](https://user-images.githubusercontent.com/11378439/73016882-bd8e0980-3dec-11ea-9b24-cb25cf4d216c.png)

Reply:
🎉 yay!